python/test/t_PointToFieldFunctionalChaosAlgorithm_std.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+# ot.Log.Show(ot.Log.INFO)
+
+# Time grid parameters
+T = 3.0
+NT = 32
+mesh = ot.RegularGrid(0.0, T / NT, NT)
+
+g = ot.SymbolicFunction(["t", "x1", "x2", "x3", "x4"],
+                        ["x1+x2*sin(t)", "x2+x3*cos(t)", "x4*t"])
+f = ot.VertexValuePointToFieldFunction(g, mesh)
+
+X = ot.Normal(4)
+
+N = 1000
+x = X.getSample(N)
+y = f(x)
```

Comment:
You see that this code is not compliant with PEP8. The upper case variable `X` is highlighted because it is expected that this is a class name.

![image](https://github.com/openturns/openturns/assets/31351465/98949ef6-9635-4cb5-8a92-5a51f2c02742)


```suggestion
inputDistribution = ot.Normal(4)
sampleSize = 1000
inputSample = inputDistribution.getSample(sampleSize)
outputSample = physicalModel(inputSample)
```